### PR TITLE
 making the blogs through markdown add tags and posts

### PR DESCRIPTION
**Pull request by LAEEQ18**
on Media Tab click on Tags that are are nested under categories in sidebar.
also add recent posts.

### CODE REVIEW

File: src/templates/tags-page.js [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import React from 'react'
+import styled from 'styled-components'
+import _ from 'lodash'
+import { Layout } from '../components/core/Layout'
+import SEO from '../components/misc/Seo'
+import { TagLink } from '../components/misc/Post'
+import { Card, CardBody } from '../components/core/Sidebar'
+
+const tagsPage = ({ pageContext }) => {
```
Comment (arif-hanif) [inline src/templates/tags-page.js:9]: components should always be PascalCase; TagsPage

File: src/templates/tag-post.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+            slug
+          }
+          excerpt
+        }
+      }
+    }
+  }
+`
+
+const tagPosts = ({ data, pageContext }) => {
```
Comment (arif-hanif) [inline src/templates/tag-post.js:40]: components should always be PascalCase; TagPosts

File: src/pages/media.js [hunk 3]
```diff
@@ -1,20 +1,89 @@
...
+          excerpt
+        }
+      }
+      totalCount
+    }
+  }
+`
+
+const Media = ({ data }) => {
+  console.log(data)
```
Comment (arif-hanif) [inline src/pages/media.js:42]: remove console.log, these should not be checked in

**Review by arif-hanif — CHANGES_REQUESTED** [review]

File: src/components/misc/PaginationLinks.js [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+        </PaginationItem>
+      )}
+    </Pagination>
+  )
+}
+PaginationLinks.propTypes = {
+  currentPage: PropTypes.number,
+  numberOfPages: PropTypes.number,
+}
+const Pagination = styled.ul`
```
Comment (LAEEQ18) [inline src/components/misc/PaginationLinks.js:54]: the code committed below is the styled components that i make for PaginationItem,& PaginationLink that i import from react strap but components are not stylled well kindly guide me about it, that how i make it well performed pagination bar

File: src/components/core/Sidebar.js [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+`
+
+const Sidebar = () => (
+  <SidebarWrapper>
+    <Card>
+      <CardBody>
+        <CardTitle> Categories </CardTitle>
+        <TagLink to="/tags/">
+          <TagHeading>TAGS</TagHeading>
+        </TagLink>
```
Comment (arif-hanif) [inline src/components/core/Sidebar.js:43]: List tags here do not go to a separate page

File: src/components/core/Sidebar.js [hunk 6]
```diff
@@ -0,0 +1,243 @@
...
+              type="email"
+              className="form-control"
+              placeholder="Your email address.."
+              name="email"
+            />
+          </FormGroup>
+          <Button>Subscribe</Button>
+        </Form>
+      </CardBody>
+    </Card>
```
Comment (arif-hanif) [inline src/components/core/Sidebar.js:61]: Make this a component and remove from page for now, not implemented right now

File: src/components/core/Sidebar.js [hunk 7]
```diff
@@ -0,0 +1,243 @@
...
+    <Card>
+      <CardBody>
+        <CardTitle>Advertisement</CardTitle>
+        <img
+          src="https://via.placeholder.com/320x200"
+          alt="Advert"
+          style={{ width: '100%' }}
+        />
+      </CardBody>
+    </Card>
```
Comment (arif-hanif) [inline src/components/core/Sidebar.js:71]: Make this a component and remove from page for now, not implemented right now

File: src/components/core/Sidebar.js [hunk 8]
```diff
@@ -0,0 +1,243 @@
...
+            </RecentPost>
+          )}
+        />
+      </CardBody>
+    </Card>
+  </SidebarWrapper>
+)
+// Sidebar.propTypes = {
+//   data: PropTypes.object,
+// }
```
Comment (arif-hanif) [inline src/components/core/Sidebar.js:105]: Remove not used code

**Review by arif-hanif — CHANGES_REQUESTED** [review]

**Pull request closed without merging**